pom.xml

```diff
@@ -45,7 +52,35 @@
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.21</version>
         </dependency>
+
+        <dependency>
+            <groupId>javax.servlet</groupId>
```

Comment:
I'm not sure if we have to keep this dependencies (jsp, jstp, servlets)

Reply:
It's dependency javax.servlet-api, we need it for work with Servlets AbstractAnnotationConfigDispatcherServletInitializer